static/src/javascripts/boot.js

```diff
@@ -72,14 +72,28 @@ const go = () => {
             });
         }
 
+        const useStandaloneBundle = window.guardian.config.tests.standaloneCommercialBundleVariant === "variant";
+        const commercialBundle = useStandaloneBundle
```

Comment:
this will trigger the fetch immediately. you might want to wrap it in a function so you can orchestrate it all below?

Reply:
I think I’ve done it properly, now!